### DISCUSSION TITLE
Olarm integration only showing 1 device, 1 entity

### DISCUSSION THREAD

**Discussion by JonnyLewis** [rainepretorius/olarm-ha-integration#116] [Q&A] [answered]
Good day my Install is connecting to Olarm but only seeing 1 entity, please can anyone assist. 

<img width="657" alt="Screenshot 2024-10-01 at 14 24 38" src="https://github.com/user-attachments/assets/1afbd08d-a0e6-4821-b0f2-bc8f077c9fbf">

<img width="483" alt="Screenshot 2024-10-01 at 14 26 05" src="https://github.com/user-attachments/assets/433d19ad-53fc-4baf-8881-7538650a5805">

**Comment by JonnyLewis** [comment, marked as answer]
Self resolved after multiple updates

**Reply by boereworswizard** [reply]
Does this integration still work for you?

**Comment by D34THSTALKER** [comment]
Mine doing the same, got 2 devices and 2 entities. tried removing and re installing but nothing seems to work.

**Comment by rainepretorius** [comment]
More than one Olarm connected to the integration does not work due to Olarm restricting third-party integration.

**Comment by D34THSTALKER** [comment]
Eish man, thanks, i was breaking my head trying to figure it out. any chance you are familiar with the paradox alarm interface add on? running my ip150 straight to ha and need some advise but this isnt the correct topic to discuss under

**Reply by rainepretorius** [reply]
pop me an email at: raine.pretorius1@gmail.com
